README.md

```diff
@@ -39,7 +39,7 @@ layer on which more complex systems can be constructed.
 
 ## Documentation
 
-* [Documentation Home](docs/index.md)
+* [Documentation Home](docs/sphinx/source/index.md)
```

Comment:
```suggestion
* [Documentation Home](https://foundationdb.github.io/fdb-record-layer/)
```

It's possible we should update this to our documentation site on GitHub Pages, but regardless, having a 404'ingredient link was definitely wrong.